torch/nested/_internal/ops.py

```diff
@@ -46,16 +47,30 @@ def _wrap_jagged_dims(ndim, dims, op_name):
     # map to inner dims so we can print a nicer error message.
     zero_in_dims = 0 in wrapped_dims
     one_in_dims = 1 in wrapped_dims
```

Comment:
I realize this is a preexisting condition, but imo it'd be better to have conceptual names here (e.g. `operates_on_ragged`, `operates_on_batch`). It looks like the names here hide the fact that the logic doesn't take into account when the NJT's ragged_idx is not 1.

not necessarily your problem to fix, but if you wanted to refactor this slightly as you expand it, I wouldn't frown upon it :)

Reply:
completely agree with this! i was planning on adding a new diff to refactor this (so as to not both add new logic and change existing variable names in one pr), but i also think it makes more sense to refactor as i go. thanks!